test-core.ini

```diff
@@ -140,7 +140,7 @@ level = WARNING
 
 [handler_console]
 class = StreamHandler
-args = (sys.stdout,)
+args = (sys.stderr,)
```

Comment:
this part seems obviously correct to me, if we make this 1-line change and update the docs to add `2> /dev/null` to the command would that be good enough?

Reply:
Good enough, sure.